examples/apps/colorapp/ecs/create-task-defs.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# set -ex
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+stack_output=$(aws --profile "${AWS_PROFILE}" --region "${AWS_REGION}" \
+    cloudformation describe-stacks --stack-name "${ENVIRONMENT_NAME}-ecs-cluster" \
+    | jq '.Stacks[].Outputs[]')
+
+task_role_arn=($(echo $stack_output \
+    | jq -r 'select(.OutputKey == "TaskIamRoleArn") | .OutputValue'))
+
+execution_role_arn=($(echo $stack_output \
+    | jq -r 'select(.OutputKey == "TaskExecutionIamRoleArn") | .OutputValue'))
+
+ecs_service_log_group=($(echo $stack_output \
+    | jq -r 'select(.OutputKey == "ECSServiceLogGroup") | .OutputValue'))
+
+envoy_log_level="debug"
+
+# Color Gateway Task Definition
+envoy_container_json=$(jq -n \
+    --arg ENVOY_IMAGE $ENVOY_IMAGE \
+    --arg VIRTUAL_NODE "mesh/$MESH_NAME/virtualNode/colorgateway-vn" \
+    --arg APPMESH_XDS_ENDPOINT "${APPMESH_XDS_ENDPOINT}" \
+    --arg ENVOY_LOG_LEVEL $envoy_log_level \
+    --arg ECS_SERVICE_LOG_GROUP $ecs_service_log_group \
+    --arg AWS_REGION $AWS_REGION \
+    --arg AWS_LOG_STREAM_PREFIX_ENVOY "colorgateway-envoy" \
+    -f "${DIR}/envoy-container.json")
+task_def_json=$(jq -n \
+    --arg NAME "ColorGateway" \
```

Comment:
It is better to qualify the name with APPMESH_STAGE to avoid conflict.

Reply:
Prefixed all the task def names with `ENVIRONMENT_NAME`